index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <title>Project Media</title>
+</head>
+<body>
+    <div id="row-1">
+        <div id="navigation-bar">
+            <input type="text" placeholder="Buscar" size="40">
+        </div>
+    </div>
```

Comment:
[REQUIRED]
- [ ] Please use semantic tags over `div` tags when appropriate. :writing_hand: 
- [ ] Please use `header` tag here. You can wrap the navigation bar with the `header` tag. :writing_hand: 
- [ ] Please use `nav` tag for your navigation bar. :writing_hand:

Reply:
Kindly explain why you suggested these changes, it might be obvious to you but not to them 👍 😄

Reply:
[REQUIRED]
- [ ] Please use semantic tags over `div` tags when appropriate. Semantic tags more accurately describe the content that is on the page. :writing_hand: 
- [ ] Please use `header` tag here. You can wrap the navigation bar with the `header` tag. The `header` element represents a group of introductory or navigational aids. :writing_hand: 
- [ ] Please use `nav` tag for your navigation bar. The `nav` element represents a section of a page that links to other pages or to parts within the page. :writing_hand: